README.md

```diff
@@ -149,50 +148,53 @@ The first round airdrop will start at around `2023-03-15-06:00 UTC`, and the sec
 }
 ```
 
+The airdrop has two rounds: In the first round, all participants will go through the same flow, and the airdrop will start at around `2023-03-15 06:00 UTC`. In the second round, all participants will go through different flows, and the airdrop will start at around `2023-03-16 06:00 UTC`. Before that, participants can prepare their strategies. Please ensure that you have been added to the gon-evidence repository and that your information is correct, otherwise you may not receive the airdrop!
+
 **Rules:**
-- The `flow` can be queryied with `flow-id` [here](https://github.com/game-of-nfts/gon-testnets/blob/main/doc/flow-table.md). 
-- The `start_height` is the block height on IRISnet. Any transfer prior to the `start_height` will be considered invalid.
-- Participants must use their own account addresses provided at the registration stage.
-- The `last_owner` is an official address on IRISnet. After finishing the interchain NFT transfer, every participant transfer the NFT to `last_owner` then complete the race.
-- Considering the time zone difference, the completion time is calculated by the difference in height between the last NFT transfer (to the `last_owner`) and the first interchain NFT transfer on IRISnet.
+
+- The `start_height` is the block height on IRISnet. Any transfer prior to the `start_height` will be considered invalid, which means your first Interchain NFT Transfer must happen after that height.
+- The `last_owner` is an official address on IRISnet, and is the final recipient of the NFT in this game. 
+- Considering the time zone difference, the completion time is calculated by the **difference** in height between the last NFT transfer (to the `last_owner`) and the first Interchain NFT transfer (not `start_height`) on IRISnet.
 - The participant with the smallest difference in block height will rank higher, and in case of a tie, the ranking will be based on the height of the last transfer, with the participant having the lower height ranking higher.
-- The end of round 3 will be at `2023-03-17-23:59:59 UTC`. The height of evidence will be considered invalid if completed after the corresponding block height.
+- Participants must use their own account addresses provided at the registration stage.
+- The third round of the competition will conclude at `2023-03-17 23:59:59 UTC`. After this, you can still complete B1 and B2, but your race rank will not be counted, and you will not be eligible to win B3 and B4.
+- The top 10 participants in each airdrop round will be awarded B3 and B4 points.
```

Comment:
I saw some people asking about B3 and B4 since it is not in the evidence.xslx, maybe point out that we are not supposed to fill in anything for B3 and B4 ourselves?

Reply:
From B1 and B2 evidence, we calculate your difference in height. Then we rank all these differences and pick up only the top 10 participants as the winners in B3 and B4.